rest-tests/src/test/java/io/syndesis/qe/rest/tests/publicapi/EnvironmentsPublicApiSteps.java

```diff
@@ -0,0 +1,50 @@
+package io.syndesis.qe.rest.tests.publicapi;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.endpoints.publicendpoint.EnvironmentsPublicEndpoint;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class EnvironmentsPublicApiSteps {
+
+    @Autowired
+    private EnvironmentsPublicEndpoint environmentsEndpoint;
+
+    @When("check that number of tags is (\\w+)$")
+    public void checkNumberOfTags(int numberOfTags) throws IOException {
+        List<String> list = getAllTags();
+        assertThat(list.size()).isEqualTo(numberOfTags);
+    }
+
+    @When("check that tag with name ([^\"]*) is in the tag list$")
+    public void checkTagExist(String tagName) throws IOException {
+        List<String> list = getAllTags();
+        assertThat(list).contains(tagName);
+    }
+
+    @When("check that tag with name ([^\"]*) is not in the tag list$")
+    public void checkTagNotExist(String tagName) throws IOException {
+        List<String> list = getAllTags();
+        assertThat(list).doesNotContain(tagName);
+    }
+
+    private List<String> getAllTags() throws IOException {
+        return environmentsEndpoint.getAllEnvironments();
+    }
+
+    @When("update tag with name ([^\"]*) to ([^\"]*)$")
+    public void updateTag(String tagName, String newTagName) {
+        environmentsEndpoint.renameEnvironment(tagName, newTagName);
+    }
+
+    @When("delete tag with name ([^\"]*)$")
+    public void deleteTag(String tagName) {
+        environmentsEndpoint.deleteEnvironment(tagName);
+    }
+}
```

Comment:
add an empty new line

Reply:
I don't know why my Idea doing this, and checkstyle doesn't warn me about it. I have to change something in Settings for this empty new line and star imports.

Reply:
I'll talk to tomas p. if we should tighten the checkstyle a bit, because currently we have the same configuration as engineering and it has only a very basic set of rules